app-games/src/main/java/com/aptoide/android/aptoidegames/di/RepositoryModule.kt

```diff
@@ -82,6 +82,12 @@ class RepositoryModule {
   fun providesAcceptLanguage(aptoideGetAcceptLanguage: AptoideGetHeaders): GetAcceptLanguage =
     aptoideGetAcceptLanguage
 
+  @Provides
+  @Singleton
+  fun providesPaymentsGetUserAgent(
```

Comment:
And what do we need this for? 🤔 
The rest client params already set with `restClientInjectParams = agGetUserAgent` and work just fine...